sdk/communication/tests.yml

```diff
@@ -1,20 +1,9 @@
 trigger: none
 
-parameters:
-  - name: TestPackagesEnabled # by overriding this parameter, developers can control what tests to run
-    displayName: Test Enabled
-    type: string
-    default: all # run live/sample tests for all packages by default
-    values:
-      - all # run live/sample tests for all packages
-      - chat # run live/sample tests for chat package only
-      - identity # run live/sample tests for identity package only
-      - phonenumbers # run live/sample tests for phonenumber package only
-      - sms # run live/sample tests for sms package only
-
 extends:
   template: ../../eng/pipelines/templates/stages/archetype-sdk-tests.yml
   parameters:
+    Project: Azure.Communication.Chat
```

Comment:
I assume this is for testing until I build out the pipeline for the sub-directory tests.yml?